testing/test_package_docs/ex/Animal-class.html

```diff
@@ -30,7 +30,7 @@
           <ol class="breadcrumbs gt-separated dark hidden-xs">
             <li><a href="index.html">test_package</a></li>
             <li><a href="ex/ex-library.html">ex</a></li>
-            <li class="self-crumb">Animal</li>
+            <li class="self-crumb">enum Animal</li>
```

Comment:
I notice we used to use a span class for the kind but don't anymore, is that intentional?